src/adr/0012-jmap-partial-reads.md

```diff
@@ -0,0 +1,39 @@
+# 12. Projections for JMAP Messages
+
+Date: 2019-10-09
+
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+JMAP core RFC8620 requires that the server responds only properties requested by the client.
+
+James currently computes all of the properties regardless of their cost, and if it had been asked by the client.
+
+Clearly we can save some latencies and resources by avoiding reading/computing expensive properties that had not been explicitly requested by the client.
+
+This is furthermore an opportunity to conform to /get behavior of released RFC-8620
```

Comment:
how?

Reply:
This comment is irrelevant as we already conform to it.